### DISCUSSION TITLE
Cannot search or follow pixefed users. How do I follow Pixelfed users from my mastodon site?

### DISCUSSION THREAD

**Discussion by jsailo** [mastodon/mastodon#19182] [Q&A]
When I Search **MarkBaker.Photo@pixelfed.social**, I get zero results on my site. But on another Mastodon site it is showing up. Is there a separate configuration for pixelfed? If so, could someone please share?

![image](https://user-images.githubusercontent.com/35369823/190840040-2eea08ff-bfd8-430e-ab65-5eb5285aa08a.png)

**Comment by aluitink** [comment]


**Reply by Gargron** [reply]
The leading `@` makes no difference.
